akka-http-core/src/main/scala/akka/http/model/parser/HeaderParser.scala

```diff
@@ -99,7 +99,7 @@ object HeaderParser {
     "range",
     "proxy-authenticate",
     "proxy-authorization",
-    "range",
```

Comment:
Yes, right.
The sorting is not actually a requirement here but I wanted to uphold it for clearity, but failed.
Will fix.